Add config flow for LG Netcast (#30110)
home-assistant/home-assistant.github.io: source/_integrations/lg_netcast.markdown

The `lg_netcast` platform allows you to control a LG Smart TV running NetCast 3.0 (LG Smart TV models released in 2012) and NetCast 4.0 (LG Smart TV models released in 2013). For the new LG WebOS TV's use the [webostv](/integrations/webostv#media-player) platform.

{% include integrations/config_flow.md %}

## Turn on action

> Home Assistant is able to turn on an LG Netcast TV is you specify an action, provided by an {% term integration %} like [HDMI-CEC](/integrations/hdmi_cec/) or [WakeOnLan](/integrations/wake_on_lan/).

Reviewer: ```suggestion
Home Assistant can turn on an LG Netcast TV if you specify an action provided by an {% term integration %} like [HDMI-CEC](/integrations/hdmi_cec/) or [WakeOnLan](/integrations/wake_on_lan/).
```

> You can create an automation from the user interface, from the device create a new automation and select the **Device is requested to turn on** automation.

Reviewer: ```suggestion
1. To create an automation, go to {% my integrations title="**Settings** > **Devices & services**" %} and open the device page. 
2. Under **Automations**, select the + icon to create an automation with that device.
3. In the dialog, select the **Device is requested to turn on** automation.

```

Automations can also be created using an automation action:

> The example below shows how you can use the `turn_on_action` the [`wake_on_lan` integration](/integrations/wake_on_lan/).

Reviewer: ```suggestion
The example below shows how you can use the `turn_on_action` with the [`wake_on_lan` integration](/integrations/wake_on_lan/).
```

```yaml
# Example configuration.yaml entry
wake_on_lan: # enables `wake_on_lan` integration

# Enables the `lg_netcast` media player
automation:
  - alias: "Turn On Living Room TV with WakeOnLan"
    trigger:
      - platform: lg_netcast.turn_on
        entity_id: media_player.lg_netcast_smart_tv
    action:
      - service: wake_on_lan.send_magic_packet
        data:
          mac: AA-BB-CC-DD-EE-FF
          broadcast_address: 11.22.33.44
```

> Any other [actions](/docks/automation/action/) to power on the device can be configured.

Reviewer 2: Typo in URL:

```suggestion
Any other [actions](/docs/automation/action/) to power on the device can be configured.
```
